### DISCUSSION TITLE
Consider new repository name

### DISCUSSION THREAD

**Discussion by seiya0914** [monstar-lab-oss/ml-frontend#688] [Ideas]
## Description

Consider new repository name, because the scope is now not only for react.js.

Here are my thoughts:
* web-front-boilerplate
* web-front-starter-kit 
* front-end-kit

**Comment by ptrkdan** [comment]
`web-front-starter-kit` sounds good to me. IMO, It sounds more all-purpose and extendable than "boilerplate".

**Comment by jinmayamashita** [comment]
`web-front-end-kit` looks good.

**Comment by jinmayamashita** [comment]
I came up with an easy one, how about `frontend-code` ? 
Maybe as a package name, not even registered in [npm](https://www.npmjs.com/search?q=frontend-code) yet.

FYI @G-N555 @ptrkdan @seiya0914

**Reply by ptrkdan** [reply]
One thing I'm concerned about is if it's too generic, people who find it may have difficulties understanding its purpose or its scope. :thinking:

**Reply by jinmayamashita** [reply]
It's alright. While few people may understand the meanings behind names like Next.js, Vue, or React, I believe there are many who understand the functionalities that these packages possess 😄

If that's the case, I think names like `mon-frontend` or `gwf` would be fine as well.

**Comment by hhofner** [comment]
"mon-frontend" (my frontend)

**Reply by jinmayamashita** [reply]
What does 'mon' mean? 'my'? 😄

**Reply by ptrkdan** [reply]
It would stick with the branding 😆

**Reply by jinmayamashita** [reply]
How about `ml-frontend` ?

**Comment by ptrkdan** [comment]
For reference, here are other ideas that we've discussed:
- @jinmayamashita `codeventure` `codecook` `codeup`
- @ptrkdan `gwf` (generate-web-frontend) 
  - _Though sometimes hard to understand, I personally think abbreviations are cool._

**Comment by jinmayamashita** [comment]
FYI @ptrkdan 
Thank you, everyone.
The name has been decided and applied, so I will close the discussion.
